src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CallVariantsFromAlignedContigsSAMSpark.java

```diff
@@ -72,7 +64,7 @@ public ReadFilter makeReadFilter() {
     protected void runTool(final JavaSparkContext ctx) {
 
         final JavaRDD<Iterable<GATKRead>> alignmentsGroupedByName = getReads().mapToPair(r -> new Tuple2<>(new Tuple2<>(r.getName(), r.getName()), r)).groupByKey().map(Tuple2::_2);
```

Comment:
Actually, we could just insist that the input SAM be queryname sorted -- then we don't have to grouping at all.